src/constants.ts

```diff
@@ -0,0 +1,48 @@
+export const emailRegex = () =>
+  /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/i;
+
+export const formatTimestampToFormDate = (value: number) => {
+  const d = new Date(value);
+  const year = d.getFullYear();
+  const month = d.getMonth() + 1;
+  const date = d.getDate();
+  return `${year}-${month < 10 ? `0${month}` : month}-${
+    date < 10 ? `0${date}` : date
+  }`;
+};
+
+export const MONTHS = [
+  "Jan",
+  "Feb",
+  "Mar",
+  "Apr",
+  "May",
+  "Jun",
+  "Jul",
+  "Aug",
+  "Sep",
+  "Oct",
+  "Nov",
+  "Dec",
+];
+
```

Comment:
I think the Intl.DateTimeFormat is perhaps most fitting for the following one as well, since you won't need to declare the months array above and you can handle any i18n need in the future.